ceph.spec.in

```diff
@@ -703,6 +704,7 @@ Recommends:     python%{python3_pkgversion}-saml
 %if 0%{?fedora} || 0%{?rhel} <= 8
 Recommends:     python%{python3_pkgversion}-grpcio
 Recommends:     python%{python3_pkgversion}-grpcio-tools
+Recommends:     python%{python3_pkgversion}-jmespath
```

Comment:
I remember I added it section as a precautionary since grpcio was not there in epel8. But its since been there now. And I saw tha jmespath is also there in epel8 so we dont need this one here. The Requires the top should cover for it.

Reply:
removed, thanks